README.md

```diff
@@ -1,9 +1,10 @@
 # Computer Vision Solution Templates
-Here you will find a set of templates that illustrates how to solve some common Computer Vision problems through Microsoft Computer Vision and Custom Vision Services.
+
+Here you'll find templates that illustrate how to use Azure's Computer Vision and Custom Vision services to implement some common digital asset management (DAM) scenarios.
```

Comment:
"some common computer vision scenarios". The DAM case is just one of the scenarios we cover here.

Reply:
updated